jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-repo.yaml

```diff
@@ -194,6 +194,12 @@
         job-name: ci-kubernetes-node-docker-benchmark
         repo-name: k8s.io/kubernetes
         timeout: 90
+    - kubernetes-node-cos-docker-validation:  
+        branch: master
+        frequency: '@hourly'
+        job-name: ci-kubernetes-node-cos-docker-validation
```

Comment:
I don't see any doc saying that it is required. But almost all jobs start with `ci-kubernetes` (others start with `pull-kubernetes` and `periodic-kubernetes`). So I think it is an implicit agreement for all CI test jobs.